features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/processor/RecordedDataProcessor.kt

```diff
@@ -48,6 +48,7 @@ internal class RecordedDataProcessor(
             rumContext = item.rumContextData.newRumContext,
             touchData = item.touchData
         )
+        prevRumContext = item.rumContextData.newRumContext
```

Comment:
if you add this here and by any chance you have a touch record that is send before a snapshot record when the view is activity/fragment is opened, this will break your logic as you do not send the `isNewView` records in the `handleTouchRecords`

Reply:
These fixes will require tests to cover all cases here